app/views/layouts/application.html.slim

```diff
@@ -35,6 +35,8 @@ html lang="#{I18n.locale}"
 
     - if Figaro.env.google_analytics_key.present?
       = render 'shared/google_analytics/page_tracking'
+    - if Figaro.env.newrelic_browser_key.present?
```

Comment:
Yes. the browser key is used by the browser instrumentation and is publicly viewable, unlike our license key. the browser key would normally be generated by the New Relic agent, however, this manual injection requires us to configure and set the browser key/app_id